### PR TITLE
Add hello-app binary for GKE + CDN examples

### PR DESCRIPTION
**Pull request by rramkumar1**
/assign @ahmetb 

FYI @steveperry-53 

Couple Notes:

1. I refrained from adding manifests/ for this application since they would essentially be copies of the ones in hello-app/manifests. If they are necessary, I can add them.

2. In the README, I did not link to any existing tutorials which use the app since they don't exist yet.

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/kubernetes-engine-samples/87)

### CODE REVIEW

**Comment by ahmetb** [comment]
LGTM
made some in browser edits so CLA bot freaked out.

File: hello-app-cdn/main.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+// hello responds to the request with a plain-text "Hello, world" message.
+// It also returns the appropriate headers to enable caching w/ the GCP CDN feature.
+func hello(w http.ResponseWriter, r *http.Request) {
+	log.Printf("Serving request: %s", r.URL.Path)
+	host, _ := os.Hostname()
+	fmt.Fprintf(w, "Hello, world!\n")
+	fmt.Fprintf(w, "Version: 1.0.0\n")
+	fmt.Fprintf(w, "Hostname: %s\n", host)
+	fmt.Fprintf(w, "Cache-Control: max-age=86400,public")
```
Comment (ahmetb) [inline hello-app-cdn/main.go]: I think this is supposed to be header @rramkumar1. 😄

**Comment by ahmetb** [comment]
Image now available at `gcr.io/google-samples/hello-app-cdn:1.0` as I added to the README.md.

But I realized there's a mistake that you don't write a header, you write the text into the response body. :)

```
$ docker run --rm -i -t gcr.io/google-samples/hello-app-cdn:1.0
Unable to find image 'gcr.io/google-samples/hello-app-cdn:1.0' locally
1.0: Pulling from google-samples/hello-app-cdn
cd784148e348: Already exists
9012bd326f57: Pull complete
Digest: sha256:ab2d0511409a2dd4a805b1e63bd08bd91f8995033d73e612a0396668420239b9
Status: Downloaded newer image for gcr.io/google-samples/hello-app-cdn:1.0

2019/01/25 00:34:04 Server listening on port 8080
```

```
curl  -v localhost:8080
* Rebuilt URL to: localhost:8080/
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> GET / HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.54.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Date: Fri, 25 Jan 2019 00:35:44 GMT
< Content-Length: 87
< Content-Type: text/plain; charset=utf-8
<
Hello, world!
Version: 1.0.0
Hostname: a4a846bfcd17
* Connection #0 to host localhost left intact
Cache-Control: max-age=86400,public%
```

File: hello-app-cdn/main.go (hunk 1 above)
Comment (rramkumar1) [reply hello-app-cdn/main.go]: Oops fixed.

**Comment by ahmetb** [comment]
Repushed `gcr.io/google-samples/hello-app-cdn:1.0`!

**Pull request merged**